tests/test_e2e_32_of_lldp.py

```diff
@@ -0,0 +1,118 @@
+
+        polling_time = 5
+
+        interface_id = "00:00:00:00:00:00:00:01:1"
+    
+        time.sleep(polling_time)
+
+        # GET topology with the interface ensuring that it's disabled
+        api_url = KYTOS_API + '/topology/v3/interfaces' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert 'looped' in data['interfaces'][interface_id]['metadata']
+        assert data['interfaces'][interface_id]['enabled'] == False
+
+
+    def test_010_lldp_ignored_loops(self):
+        """ Test that the ports 4 and 5 are ignored. """
+
+        polling_time = 5
+
+        interface_id4 = "00:00:00:00:00:00:00:01:4"
+
+        # GET topology with the interfaces ensuring that they are enabled
+        api_url = KYTOS_API + '/topology/v3/interfaces' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert data['interfaces'][interface_id4]['enabled'] == True
+
+        # WAIT for some time, until the feature kicks
+        time.sleep(polling_time)
+
+        # GET topology with the interface ensuring that they are enabled
+        api_url = KYTOS_API + '/topology/v3/interfaces' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert data['interfaces'][interface_id4]['enabled'] == True  # [4,5] is an ignored loop
+
+
+    def test_020_reconfigure_ignored_loops(self):
+        """ Test that when we reconfigure the ignored loop with 
+        POST /api/kytos/topology/v3/switches/{{dpid}}/metadata 
+        the loops aren't ignored anymore.. """
+
+        polling_time = 5
+
+        switch = '00:00:00:00:00:00:00:01'
+        interface_id4 = "00:00:00:00:00:00:00:01:4"
+
+        # GET topology with the interfaces ensuring that they are enabled
+        api_url = KYTOS_API + '/topology/v3/interfaces' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert data['interfaces'][interface_id4]['enabled'] == True
+
+        # WAIT for some time, until the feature kicks
+        time.sleep(polling_time)
+
+        # Reconfigure the ignored loops
+        api_url = KYTOS_API + '/topology/v3/switches/%s/metadata' % switch
+        response = requests.post(api_url, json={"ignored_loops": []})
+        assert response.status_code == 201, response.text
+
+        self.restart()
+
+        # GET topology with the interface ensuring that they are enabled
+        api_url = KYTOS_API + '/topology/v3/interfaces' 
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        data = response.json()
+        assert data['interfaces'][interface_id4]['enabled'] == False # [4,5] is an ignored loop, it was reconfigured
+
```

Comment:
@gretelliz, at the moment we don't have linters running for this repo, but 7 blank lines were added here, can you remove them too since you're at it?

Reply:
Sure

Reply:
Looks good, thanks. I took the opportunity to also map a requirement to enable linters and formatters in the future in this repo https://github.com/amlight/kytos-end-to-end-tests/issues/141.